HLTrigger/Configuration/python/customizeHLTforMC.py

```diff
@@ -18,6 +18,13 @@ def customizeHLTforMC(process,_fastSim=False):
     process.hltParticleFlowRecHitHCAL.ApplyPulseDPG      = cms.bool(False)
     process.hltParticleFlowRecHitHCAL.LongShortFibre_Cut = cms.double(1000000000.0)
 
+  # customise hltHbhereco to use the Method 3 parametrisation for Monte Carlo (see #11091)
+  if 'hltHbhereco' in process.__dict__:
```

Comment:
If thsi is for MC only and not for real data, you can not use customizeHLTforMC.py.
Data vs MC differences must be solved via the globaltag mechanism.